OrbitClientGgp/ClientGgp.cpp

```diff
@@ -79,9 +78,105 @@ bool ClientGgp::StopCapture() {
   return capture_client_->StopCapture();
 }
 
-void ClientGgp::InitCapture() {
+std::shared_ptr<Process> ClientGgp::GetOrbitProcessByPid(int32_t pid) {
+  // We retrieve the information of the process to later get the module corresponding to its binary
+  ErrorMessageOr<std::vector<ProcessInfo>> result_process_infos = process_client_->GetProcessList();
+  if (result_process_infos.has_error()) {
+    return nullptr;
+  }
+  const std::vector<ProcessInfo>& process_infos = result_process_infos.value();
+  LOG("List of processes:");
+  for (const ProcessInfo& info : process_infos) {
+    LOG("pid:%d, name:%s, path:%s, is64:%d", info.pid(), info.name(), info.full_path(),
+        info.is_64_bit());
+  }
+  auto process_it = find_if(process_infos.begin(), process_infos.end(),
+                            [&pid](const ProcessInfo& info) { return info.pid() == pid; });
+  if (process_it != process_infos.end()) {
```

Comment:
if (process_it == process_infos.end()) {
  return nullptr;
}

...

Reply:
this makes a lot of sense, will update it, thanks!

Reply:
Done